src/OVAL/probes/probe-api.c

```diff
@@ -1405,7 +1405,7 @@ SEXP_t *probe_item_create(oval_subtype_t item_subtype, probe_elmatr_t *item_attr
         bool    multiplied;
         int     value_i, multiply;
 
-        subtype_name = oval_subtype_to_str(item_subtype);
+        subtype_name = oval_subtype_get_text(item_subtype);
```

Comment:
You are true, I haven't considered this.

I have looked more into this. I have noticed that `oval_enumeration_get_text(map, subtype)` can return NULL, so if item_subtype is invalid, it can return either `NULL` or `"**INVALID**"` (both of the possibilities dependent on the reason of invalidity).

I suggest returning NULL always